### ISSUE TITLE
Alcatel GO FLIP

### ISSUE THREAD

**Issue by sorccu** [openstf/stf#587]
**What is the issue or idea you have?**

When connected to STF, Alcatel GO FLIP is in a permanent prepare loop. This seems to be caused by a modified ROM disabling app_process and therefore most internal commands not working.

It's unclear whether there's anything we can do about this issue, possibly not.

**Does it only happen on a specific device? Please run `adb devices -l` and paste the corresponding row.**

> product:gflip2 model:Alcatel_GO_FLIP device:gflip2

**Please provide the steps to reproduce the issue.**

Connect an Alcatel GO FLIP to STF.

**What is the expected behavior?**

Device shows up normally in STF.

**Do you see errors or warnings in the `stf local` output? If so, please paste them or the full log here.**

The cause is:

```
adb shell pm list packages
/system/bin/pm[6]: app_process: not found
```

Obviously the ROM has been modified, probably heavily stripped down for the low-spec device. Therefore you can't even launch most executables on that device.

**Please run `stf doctor` and paste the output here.**

Not relevant.

**Comment by 0x88l** [comment]
Where is the phone manufacturer?I never hear the brand.

**Comment by 3dgiordano** [comment]
Hello everyone.

I searched for information and found that the user interface does not looks
like android and there are mentions of KaiOS created by KaiOS Technologies
Inc https://www.linkedin.com/company-beta/13291989/?pathWildcard=13291989

I think that KaiOS is based on Firefox OS

Maybe the output of this commands helps to identify the OS version

adb shell getprop
or
adb shell cat /system/build.prop

I hope it helps.
regards


2017-04-28 6:54 GMT-03:00 0x88l <notifications@github.com>:

**Comment by 3dgiordano** [comment]
Perfect!, it only remains to know if it is a FirefoxOS

I do not know anything about FirefoxOS, it is based on some data that I
found on the internet.

adb shell ls /data/b2g/mozilla/*.default/prefs.js

or running the ps version of b2g

adb shell b2g-ps




2017-04-28 10:11 GMT-03:00 Simo Kinnunen <notifications@github.com>:

**Comment by sorccu** [comment]
Good call, b2g-ps works! The other one gives permission denied. Seems like
it's Firefox OS.

**Comment by 3dgiordano** [comment]
Great!. Mystery solved. :)

A pleasure to help you.
regards

2017-04-28 10:52 GMT-03:00 Simo Kinnunen <notifications@github.com>:

> Good call, b2g-ps works! The other one gives permission denied. Seems like
> it's Firefox OS.
>
>
